activesupport/lib/active_support/i18n_railtie.rb

```diff
@@ -4,6 +4,15 @@
 
 module I18n
   class Railtie < Rails::Railtie
+    module Helpers
+      def self.watched_dirs_with_extensions(paths)
```

Comment:
This `Helpers` isn't pulling it's weight anymore. Lets just add this as a private class method in the Railtie. We control the methods there, so there's no chance of name collisions if that's what you're worried about :smile: